### PR TITLE
Partially fix loading non-Windows NE files.

### PR DESCRIPTION
**Pull request by claunia**
Fix loading a OS/2 NE with resources and an OS/2 driver.

### CODE REVIEW

**Comment by uxmal** [comment]
Thanks for your first contribution! One small detail: could you add your name (and/or) alias to the AUTHORS file in the root directory?

I will address the resource types in issue #841.

File: src/ImageLoaders/MzExe/NeImageLoader.cs [hunk 1]
```diff
@@ -407,17 +420,45 @@ public override Program Load(Address addrLoad)
...
+            switch (bTargetOs)
+            {
+                case NE_TARGETOS.Windows:
+                case NE_TARGETOS.Windows386:
+                    this.platform = cfgSvc.GetEnvironment("win16").Load(Services, arch);
+                    break;
+                case NE_TARGETOS.EuropeanDos:
+                    this.platform = cfgSvc.GetEnvironment("ms-dos").Load(Services, arch);
+                    break;
+                default:
```
Comment (uxmal) [inline src/ImageLoaders/MzExe/NeImageLoader.cs]: Commit 027904892e1a4c5e8e95b61e194d4bf241597d4d added support for the `os2` platform, you can change this to:
```C#
case NE_TARGETOS.Os2:
    this.platform = cfgSvc.GetEnvironment("os2").Load(Services, arch);
    break;
```
if you have the time.

**Comment by claunia** [comment]
Ok only missing, well, whatever are you doing with icons in the Windows side. And some testing, I still get an exception somewhere.

**Comment by uxmal** [comment]
Are you OK with merging, or do you wish me to wait until you've chased down the exception?

**Comment by claunia** [comment]
I'm ok that way you can also take a look, you know the codebase better so you may find it first. Checked what you were doing with the Windows resources I see you are adding the Bitmap header to the DIB data in Windows resource. Not needed in OS/2 as they always have the header (but it´s a different format, I also have a decoder that gets a raw array of pixels).

**Comment by uxmal** [comment]
OK I will pull it then. ¡Muchas gracias por la contribución!

**Comment by claunia** [comment]
De nada. I really need to decompile that HPOFS driver so :D My benefit, your benefit, our benefit.

**Pull request merged**